src/NLog/Internal/FileInfoExt.cs

```diff
@@ -43,19 +43,16 @@ public static DateTime GetLastWriteTimeUtc(this FileInfo fileInfo)
 #if !SILVERLIGHT
             return fileInfo.LastWriteTimeUtc;
 #else
-            return fileInfo.LastWriteTime;
+            return fileInfo.LastWriteTime.ToUniversalTime();
```

Comment:
ow yes, missed the `!`.

I think this is then OK, iif the tests could work without UTC stuff